src/components/job-common.js

```diff
@@ -161,3 +193,54 @@ export const workflowDetailsBreadcrumbSubtitle = (namespace, workspaceName, subm
     h3({ style: Style.elements.sectionHeader }, [`Workflow ${workflowId}`]),
   ]);
 };
+
+export const Breadcrumbs = ({ breadcrumbPathObjects, pageId }) => {
```

Comment:
my first thought upon seeing this was: "surely this exists somewhere else, or some other component has the name  `Breadcrumbs` already". But I looked around and couldn't find anything.

I don't have a specific suggestion - I would just be careful about naming something so generically like this, just in case! (On the other hand, it does appear to be a pretty generic component, so I'm not saying it's wrong)

Reply:
It seems we do have similar bread crumbs UX in Submission Details and Workflow Dashboard in GCP workspaces. I see that the styling and wordings are different but maybe it can be refactored to make it more generic so that it can be used here as well as in GCP version? 

![Screenshot 2023-08-07 at 12 25 31 PM](https://github.com/DataBiosphere/terra-ui/assets/16748522/758cd00c-3ee7-46bc-8b78-732fd176f38d)